### ISSUE TITLE
Radio Systems

### ISSUE THREAD

**Issue by ausshir** [UofSSpaceDesignTeam/balloon-software#4]
Some Alternatives
- Serial link for basic sensor data
- Cellular Radio

**Comment by jjball** [comment]
I am not too sure if we can use a cellular radio in the balloon, as according to this:
http://www.ecfr.gov/cgi-bin/text-idx?SID=3a6f5c68673c71cf564f7db91144bafe&mc=true&node=pt47.2.22&rgn=div5#se47.2.22_1925

Which was referenced from this website:
http://www.highaltitudescience.com/pages/tracking-a-weather-balloon

It may be illegal to use a sim card or the cellular network to track the balloon?

But some other places state that using a cell phone is a cheap alternative that works up to 5000 feet, such as in this article:
http://community.balloonchallenge.org/t/current-gps-tracker-products/668

The articles stating that using the cellular network is illegal are mainly from the US, while the ones suggesting to use them have been from the UK or South Africa so I assume it is illegal.

**Comment by ottopasuuna** [comment]
Yeah, I found that article as well and was hoping some Canadian specific regulations could be found. I found [this](https://www.tc.gc.ca/eng/civilaviation/opssvs/managementservices-referencecentre-acs-700-700-005-303.htm), but like most things it refers to airplanes specifically, and doesn't imply anything for unmanned balloons. It's probably difficult to search for the absence of laws or laws that explicitly allow the use of cellular radio at high altitude, so it may be best to look at other alternatives.

Also, after looking at the third link Jaden posted, the issue of amatuer radio licenses arises. Are we covered by launching with the Saskatoon amateur radio club, or will we need our own license if we include our own radios?

**Comment by bht822** [comment]
I found this for Arduino , it's easy to program and weighs less as well  

http://www.instructables.com/id/Make-a-High-Altitude-Balloon-Tracker-Arduino/

**Comment by ausshir** [comment]
@ottopasuuna We should be able to ask Bruce from SARC about the regulations
